chrome-habit-tracker-master/index.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+//Declare global variables
+let habitStorage = {'habits': []}; //For updating sync storage
+let actualDays; //Days in selected month for rendering
+let monthSelected = new Date().getMonth(); //Actual month selected 0-indexed
+
+document.addEventListener('DOMContentLoaded', () => {
+  chrome.storage.sync.get('habits', (habits) => {
+    //Check sync storage
+    if (habits.hasOwnProperty('habits')) Object.assign(habitStorage, habits);
+    //Chrome doesn't allow this to be put directly on index.html
+    document.getElementById('add-habit-button').addEventListener('click', showInput);
+    //Render grid
+    setGrid(new Date().getMonth());
+  });
+
+});
+
+const getDaysInMonth = (month) => { //Month passed as 1-indexed
+  return new Date(new Date().getFullYear(), month + 1, 0).getDate();
+}
+
+const createNode = (type, classGiven) => { //Shortcut for creating nodes
+  const node = document.createElement(type);
+  node.className = classGiven;
+  return node;
+}
+
+const setGrid = (month) => {
```

Comment:
you should keep code more modularized, in this case I would create one file for render functions and another for input functions